lib/blumquist.rb

```diff
@@ -2,9 +2,12 @@
 require 'active_support/core_ext/hash/indifferent_access'
 require 'json'
 require 'json-schema'
+require 'blumquist/json_pointer'
 require 'blumquist/errors'
 
 class Blumquist
+  PRIMITIVE_TYPES = %w{null boolean number string enum integer}.freeze
```

Comment:
Moved to a constant as this list is also used [here](https://github.com/moviepilot/blumquist/pull/22/files#diff-323f52f0e9e59e9550be13a6fd29dfaeR5).